src/containers/Cluster/ClusterInfo/ClusterInfo.tsx

```diff
@@ -204,20 +145,45 @@ export const ClusterInfo = ({
         ...links,
     ]);
 
+    const getContent = () => {
+        if (loading) {
+            return <ClusterInfoSkeleton />;
+        }
+
+        if (error) {
+            <ResponseError error={error} className={b('error')} />;
+        }
+
+        return <InfoViewer dots={true} info={clusterInfo} />;
+    };
+
+    const getClusterTitle = () => {
+        if (loading) {
+            return <Skeleton className={b('title-skeleton')} />;
+        }
+
+        return (
+            <EntityStatus
+                size="m"
+                status={cluster?.Overall}
+                name={cluster?.Name ?? 'Unknown cluster'}
+                className={b('title')}
+            />
+        );
+    };
+
```

Comment:
Two different skeletons for the info and for the title to include them in `ClusterInfo` layout and to enable expand button for the skeleton as well. This should enable user to hide content even if it's not loaded yet. May be helpful for big clusters, where `ClusterInfo` could load 20+ seconds, but there could be a need to to view tabs content ASAP